hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/FederationClientInterceptor.java

```diff
@@ -197,10 +196,9 @@ public void init(String userName) {
 
     numSubmitRetries =
         conf.getInt(YarnConfiguration.ROUTER_CLIENTRM_SUBMIT_RETRY,
-            YarnConfiguration.DEFAULT_ROUTER_CLIENTRM_SUBMIT_RETRY);
+        YarnConfiguration.DEFAULT_ROUTER_CLIENTRM_SUBMIT_RETRY);
```

Comment:
The indentation is weird now. Probably the cleanest is:
```
numSubmitRetries = conf.getInt(
    YarnConfiguration.ROUTER_CLIENTRM_SUBMIT_RETRY,
    YarnConfiguration.DEFAULT_ROUTER_CLIENTRM_SUBMIT_RETRY);
```

Reply:
I will Fix it.